core/sis-utility/src/main/java/org/apache/sis/util/collection/RangeSet.java

```diff
@@ -412,7 +412,7 @@ private boolean isSorted() {
         if (array == null) {
             return true;
         }
-        final boolean strict = isMinIncluded | isMaxIncluded;
+        final boolean strict = isMinIncluded || isMaxIncluded;
```

Comment:
Actually, if the compiler were generating code verbatim, the use of `|` operator would be more efficient than `||` because a bitwise `OR` operation is faster than a jump on modern processor. But I guess that in practice, optimization done by the compiler results in the same assembly code in the end.

Reply:
Well, that astonished me due to the general known logical vs. bitwise stuff.
However, I've learned something and you're completely right -
 http://javarevisited.blogspot.co.at/2015/01/difference-between-bitwsie-and-logical.html
Thank you!
I am going to undo this change.